client/src/components/Info.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { useState, useEffect } from 'react'
+import Map from './Map'
+import ReviewForm from './ReviewForm'
+
+
+export default function Info(props) {
+
+    //Restaurant info
+    const [restoObj, setRestoObj] = useState({})
+    const [id, setId] = useState()
+    const [place, setPlace] = useState("Name not Found")
+    const [address, setAddress] = useState("Address not Found")
+    const [number, setNumber] = useState("Number not Found")
+    const [hours, setHours] = useState("Hours not Found")
+    const [description, setDescription] = useState([])
+    const [reviews, setReviews] = useState([])
```

Comment:
You could narrow down your amount of state by creating one state that would contain the entire object that is pulled back from your state. Then you can access the same information by saying something like `stateObj.place` and `stateObj.address`. Doing it that way it reduces the lines of code you have for building up state but also for setting each individual one to those values to state.